WhoReacted/WhoReacted.plugin.js

```diff
@@ -245,29 +251,15 @@ module.exports = !global.ZeresPluginLibrary ? class {
                 }
             );
 
-            Reactions.forceUpdateAll();
+            ReactComponents.getComponentByName("Reactions", DiscordSelectors.Reactions.reactions).then(c => c.forceUpdateAll());
         }
 
         async findReaction() {
-            const Reactions = await this.findReactions();
-
-            const unpatch = Patcher.after(Reactions.component.prototype, "render", (thisObject, args, returnValue) => {
```

Comment:
Sometimes the Reaction component isn't found by just calling `ReactComponents.getComponentByName` usually after you have just opened discord. When you push the component it is put in a cache and then no more problems will happen when finding it.